Gems/PhysX/Code/Source/HingeJointComponent.cpp

```diff
@@ -85,4 +88,60 @@ namespace PhysX
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
     }
+
+    // AZ::Component
+    void HingeJointComponent::Activate() {
+        JointComponent::Activate();
+        PhysX::JointInterfaceRequestBus::Handler::BusConnect(this->GetEntityId());
+    }
+
+    void HingeJointComponent::Deactivate() {
+        JointComponent::Deactivate();
+        PhysX::JointInterfaceRequestBus::Handler::BusDisconnect();
+    }
+
+    physx::PxRevoluteJoint* GetPhysXNativeRevoluteJoint(const AzPhysics::JointHandle& joint_handle,
+                                                        const AzPhysics::SceneHandle& joint_scene_owner)
+    {
+        auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        const auto joint = sceneInterface->GetJointFromHandle(joint_handle,joint_scene_owner);
+        const auto type = joint->GetNativeType();
+        AZ_Assert(type == PhysX::NativeTypeIdentifiers::HingeJoint, "It is not PhysXHingeJoint");
+        physx::PxJoint *native_joint = static_cast<physx::PxJoint*>(joint->GetNativePointer());
+        physx::PxRevoluteJoint *native =native_joint->is<physx::PxRevoluteJoint>();
+        AZ_Assert(native, "It is not PxRevoluteJoint");
+        return native;
+    }
+
+    float HingeJointComponent::GetPosition(){
+        return GetPhysXNativeRevoluteJoint(m_jointSceneOwner,m_jointHandle)->getAngle();
```

Comment:
suggestion: We might want to cache the value returned be `GetPhysXNativeRevoluteJoint` so we don't have to invoke it for every `Get` call (not critical but might be a performance win depending on how expensive `GetJointFromHandle` is.

Reply:
[standard] space after the comma in parameters.`(m_jointSceneOwner, m_jointHandle)`

Same in other places. (I won't mention it again, please review the PR having this into consideration)

Reply:
I've reworked this code, this function is a protected method now, without a parameter. It is getter to PhysX native joint type with cache.